test/lib/test-server.js

```diff
@@ -0,0 +1,56 @@
+var request = require('request')
+var createApp = require('../../index.js')
+
+var portCounter = 10000
+
+module.exports = function (cb) {
+  // setup config
+  // =
+
+  var config = {
+    township: {
+      secret: 'very very not secret',
+      memdb: true,
```

Comment:
Im open to switching. Reason for this: If we avoid putting objects like a memdb in the hypercloud config, then the config could be json